applications/StructuralMechanicsApplication/custom_processes/total_structural_mass_process.cpp

```diff
@@ -56,7 +56,7 @@ void TotalStructuralMassProcess::Execute()
         }
     }
     
-    std::cout << "The total mass of the system is :" << total_mass/1000.0 << " Tn\n" << "Check variable NODAL_MASS in the process info in order to access to it in any moment" << std::endl;
+    KRATOS_INFO("TOTAL MASS") << "The total mass of the system is :" << total_mass/1000.0 << " Tn\n" << "Check variable NODAL_MASS in the process info in order to access to it in any moment" << std::endl;
```

Comment:
I would remove the description sentence. Note that logger in normal way writes the labe : values. So the above statement would output:

```
TOTAL MASS : The total mass of the system is : 2497.234
```
And in table_output it will try to write the sentence in Total Mass column which leave you without space for writing the value.

I would also avoid all capital labels to camel case title leaving the all capital for by default labels.
```c++
KRATOS_INFO("Total Mass") << total_mass/1000.0 << " Tn" << std::endl;
KRATOS_DETAIL("Hint")  << "Check variable NODAL_MASS in the process info in order to access to it in any moment" << std::endl;
```

Reply:
@pooyan-dadvand didn't we say to use a keyword arguement for the "TOTAL MASS"? (i agree about using camelcase, it is just to highlight which part of the message i refer to)

Reply:
One thing "KRATOS_DETAIL" does not exist